e2e/packages/client-vanilla/package.json

```diff
@@ -34,6 +34,6 @@
   "devDependencies": {
     "rimraf": "^3.0.2",
     "vite": "^4.2.1",
-    "vitest": "0.31.4"
+    "vitest": "^1.0.4"
   }
-}
+}
```

Comment:
hmm we should figure out why this trailing empty line differs as it prob points to an inconsistent linter or formatter

did you edit these files manually or use something like `pnpm upgrade`?

Reply:
manually 😬 couldn't figure out how to specify the version with `pnpm ugprade`

Reply:
I enabled **Insert Final Newline** in my VSCode and the formatting is correct now